src/middleware/auth.middleware.js

```diff
@@ -0,0 +1,22 @@
+import { jwtService } from '../services/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  const userData = jwtService.validateAccessToken(token);
```

Comment:
Consider handling potential errors from 'jwtService.validateAccessToken(token)' to avoid server crashes if the token validation fails unexpectedly.